api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java

```diff
@@ -320,6 +320,10 @@ public class UserVmResponse extends BaseResponseWithTagInformation implements Co
     @Param(description = "true if vm contains XS/VMWare tools inorder to support dynamic scaling of VM cpu/memory.")
     private Boolean isDynamicallyScalable;
 
+    @SerializedName(ApiConstants.DELETE_PROTECTION)
+    @Param(description = "true if vm has delete protection.")
```

Comment:
```suggestion
    @Param(description = "true if vm has delete protection.", since = "4.20.0")
```

add since if this is targeted for 4.20